<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayer</class>
 <widget class="QMainWindow" name="MusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>140</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FrutaGroove Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainRes.qrc">
    <normaloff>:/images/resources/fgpNewLogo.png</normaloff>:/images/resources/fgpNewLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>356</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QGridLayout" name="mediaLayout">
       <item row="1" column="0">
        <layout class="QGridLayout" name="controlsLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="rewindButton">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="mainRes.qrc">
             <normaloff>:/images/resources/buttonRewindLight.png</normaloff>:/images/resources/buttonRewindLight.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="stopButton">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="mainRes.qrc">
             <normaloff>:/images/resources/buttonStopLight.png</normaloff>:/images/resources/buttonStopLight.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="fowardButton">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="mainRes.qrc">
             <normaloff>:/images/resources/buttonFowardLight.png</normaloff>:/images/resources/buttonFowardLight.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="volumeButton">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="mainRes.qrc">
             <normaloff>:/images/resources/volumeLevelLight.png</normaloff>:/images/resources/volumeLevelLight.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="playButton">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="mainRes.qrc">
             <normaloff>:/images/resources/buttonPlayLight.png</normaloff>:/images/resources/buttonPlayLight.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QSlider" name="volumeSlider">
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QSlider" name="progressSlider">
         <property name="minimumSize">
          <size>
           <width>352</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>352</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="songName">
       <property name="maximumSize">
        <size>
         <width>367</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Song"/>
    <addaction name="actionOpen_Playlist"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionCheck_for_Updates"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPlaylists"/>
    <addaction name="actionVisualizer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen_Song">
   <property name="text">
    <string>Open Song</string>
   </property>
  </action>
  <action name="actionOpen_Playlist">
   <property name="text">
    <string>Open Playlist</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About FGP</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
  </action>
  <action name="actionShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </action>
  <action name="actionPlaylists">
   <property name="text">
    <string>Playlist</string>
   </property>
  </action>
  <action name="actionVisualizer">
   <property name="text">
    <string>Visualizer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainRes.qrc"/>
 </resources>
 <connections/>
</ui>
